geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/security/SimpleFeatureSecurityWrapper.scala

```diff
@@ -0,0 +1,33 @@
+package org.locationtech.geomesa.utils.security
+
+import org.opengis.feature.simple.SimpleFeature
+
+import scala.language.implicitConversions
+
+/**
+ * Wraps a [[SimpleFeature]] allowing visibility to be accessed and modified.
+ * 
+ * Created by mmatz on 4/3/15.
+ */
+class SimpleFeatureSecurityWrapper(val sf: SimpleFeature) {
```

Comment:
Should make this class implicit and extend AnyVal - that way, it will not create actual objects, the calls will just be static methods. The class has to be moved inside an object.